substrate/frame/revive/src/limits.rs

```diff
@@ -46,6 +51,9 @@ pub const NUM_EVENT_TOPICS: u32 = 4;
 /// Maximum size of events (including topics) and storage values.
 pub const PAYLOAD_BYTES: u32 = 416;
 
+/// The maximum size for calldata and return data.
```

Comment:
IIRC this value matches the current practical limit on Ethereum. Might point that out here?